packages/code/src/renderingPlugin.tsx

```diff
@@ -54,6 +54,18 @@ function isNumber(item: any): item is number {
   return item != null && typeof item === 'number';
 }
 
+function isTokenObject(item: any): item is TokenObject {
+  // kind: string;
+  // children: Array<string | TokenObject>;
+  // sublanguage?: boolean;
+
+  if (item == null || typeof item !== 'object') {
+    return false;
+  }
+
+  return typeof item?.kind === 'string' && item?.children instanceof Array;
```

Comment:
why can't we first assert that item isn't null so that we can do item.kind and item.children

Reply:
That's a good point, we already do that so the optional chaining here is unnecessary.